db/data/20210127153525_set_financial_year_based_on_start_date.rb

```diff
@@ -0,0 +1,12 @@
+class SetFinancialYearBasedOnStartDate < ActiveRecord::Migration[6.0]
+  def up
+    Budget.all.each do |budget|
```

Comment:
Probably worth doing `Budget.all.find_each` here instead.

Reply:
Do we want to drop the `period_start_date` and `period_end_date` columns too?

Reply:
I deliberately skipped that. I don't 100% trust the Data Migrate gem, and would rather wait until the data migrations are done before dropping the column. Will make a card so we don't forget 👍